lib/sys-info.ts

```diff
@@ -35,15 +35,17 @@ export class SysInfo implements ISysInfo {
 	}
 
 	@exported("sysInfo")
-	public async getSystemWarnings(): Promise<string[]> {
-		const warnings: string[] = [];
+	public async getSystemWarnings(): Promise<ISystemWarning[]> {
+		const warnings: ISystemWarning[] = [];
 		const macOSWarningMessage = await this.getMacOSWarningMessage();
 		if (macOSWarningMessage) {
+			macOSWarningMessage.toString = function() { return this.message; };
 			warnings.push(macOSWarningMessage);
 		}
 
 		const nodeWarning = getNodeWarning();
 		if (nodeWarning) {
+			nodeWarning.toString = function() { return this.message; };
```

Comment:
This will not work - the current class does not have `message` property and in case I use lambda, the current `this` will be used inside it. That's why I need function.